README.md

```diff
@@ -148,6 +148,11 @@ return array(
     //
     // To disable update checks entirely, set to 'never'.
     'updateCheck' => 'daily',
+
+    // Display an additional startup message. Default is ''.
+    // You can color and style the message thanks to the Symfony Console tags.
+    // See https://symfony.com/doc/current/console/coloring.html for more details.
+    'startupMessage' => sprintf('<info>%s</>', shell_exec('uptime')),
```

Comment:
for the example, let's use a full closing tag rather than the `</>` shortcut. that one is a bit weird without context :)